gpiozero/other_devices.py

```diff
@@ -118,8 +118,11 @@ def __init__(self, sensor_file='/sys/class/thermal/thermal_zone0/temp',
             min_temp=0.0, max_temp=100.0, threshold=80.0):
         self.sensor_file = sensor_file
         super(CPUTemperature, self).__init__()
+        if min_temp >= max_temp:
+            raise ValueError('min_temp must be less than max_temp')
         self.min_temp = min_temp
         self.max_temp = max_temp
+        self._temp_range = self.max_temp - self.min_temp
```

Comment:
Can't do this - it doesn't support changing the values after init. This could be a property instead but that's overkill I think.

Reply:
Right.
I actually took it from here:
`        self._dc_range = (max_pulse_width - min_pulse_width) / frame_width

    @property
    def max_pulse_width(self):
        """
        The control pulse width corresponding to the servo's maximum position,
        measured in seconds.
        """
        return (self._dc_range * self.frame_width) + self.min_pulse_width
`

Reply:
Right. I actually took it from here

Servo init:
`self._dc_range = (max_pulse_width - min_pulse_width) / frame_width`
  max_pulse_width property:
`return (self._dc_range * self.frame_width) + self.min_pulse_width`

Reply:
But that's a read-only property there.